src/org/loklak/LoklakServer.java

```diff
@@ -543,6 +544,7 @@ private static void setServerHandler(File dataFile){
         servletHandler.addServlet(ProxyServlet.class, "/api/proxy.jpg");
         servletHandler.addServlet(ValidateServlet.class, "/api/validate.json");
         servletHandler.addServlet(GenericScraper.class, "/api/genericscraper.json");
+        servletHandler.addServlet(RSSReader.class, "/api/rssreader.json");
```

Comment:
@daminisatya I believe it's not. 
Still, let me test with a few other RSS Feeds, Shall let know...